src/checks/graphiteSpike.check.js

```diff
@@ -95,7 +95,8 @@ class GraphiteSpikeCheck extends Check {
 			})
 			.catch(err => {
 				logger.error({ event: `${logEventPrefix}_ERROR`, url: this.sampleUrl }, err);
-				this.status = status.FAILED;
+				this.status = status.ERRORED;
+				this.severity = 3;
```

Comment:
Changing the severity on error is a bit of an anti-pattern.  I can see why you've done it here, but perhaps we need a better way to distinguish better between "thing broken" and "dunno if thing is broken 🤷‍♀️ ".

Reply:
I was hoping status.ERRORED might be that thing - but it's not in the healthcheck standard anywhere so I'm not actually sure if it does anything